pages/windows/choco-pack.md

```diff
@@ -0,0 +1,15 @@
+# choco pack
+
+> Package a NuGet specification into a nupkg file.
+
+- Package a NuGet specification to a nupkg file:
+
+`choco pack {{path/to/specification}}`
+
+- Package a NuGet specification using a specific version:
```

Comment:
Does this mean the version of the package that's being packed?

Reply:
Yes, this is the version of the package that's being packed. 👍

Reply:
Ah, ok. Hrm. I wonder if there's a may to make it clearer?

What about `specifying the version of the resulting file`?

Reply:
I guess that makes it clearer. :+1: